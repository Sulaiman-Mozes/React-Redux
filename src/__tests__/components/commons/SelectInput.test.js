import React from 'react';
import { shallow } from 'enzyme';
import SelectInput from '../../../components/commons/SelectInput';
import { authors } from '../../../../tools/mockData';


describe('SelectInput Component', () => {
  it('should render the SelectInput Component', () => {
    const props = {
      name: '',
      label: '',
      onChange: jest.fn(),
      defaultOption: '',
      value: '',
      error: '',
      options: authors.map(author => ({
        value: author.id,
        text: author.name,
      })),
    };
    const wrapper = shallow(<SelectInput {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
