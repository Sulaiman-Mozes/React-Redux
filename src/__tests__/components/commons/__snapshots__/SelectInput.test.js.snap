// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectInput Component should render the SelectInput Component 1`] = `ShallowWrapper {}`;
